opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -163,11 +164,33 @@ def create(
         Returns:
             The newly-created Resource.
         """
+
         if not attributes:
             attributes = {}
-        resource = _DEFAULT_RESOURCE.merge(
-            OTELResourceDetector().detect()
+
+        resource_detectors = []
+
+        resource = _DEFAULT_RESOURCE
+
+        for resource_detector in environ.get(
+            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS, "otel"
+        ).split(","):
+
+            resource_detectors.append(
+                next(
+                    iter(
+                        entry_points(
+                            group="opentelemetry_resource_detector",
+                            name=resource_detector,
+                        )
+                    )
+                ).load()()
+            )
+
+        resource = get_aggregated_resources(
+            resource_detectors, _DEFAULT_RESOURCE
         ).merge(Resource(attributes, schema_url))
```

Comment:
This should make sure to include `OTELResourceDetector`

Reply:
Fixed